scripts/xmpro-website-scripts/scrape-xmpro-website-latestnews-config.json

```diff
@@ -0,0 +1,3 @@
+{
+    "folderPath": "docs/external-content/latest-news"
```

Comment:
@jacksonmcguirkxmpro 

[same as here](https://github.com/XMPro/public-docs/pull/6/commits/3c24a90efc3a34d8def371afdd133d9f8e1aad97#diff-5197550e5489be215d608fc6401370e7687e76d3a7886f836fafe835a77c443fR2)

TODO for all